system/handlers/admin/FormBuilder.cfc

```diff
@@ -611,6 +653,83 @@ component extends="preside.system.base.AdminHandler" {
 		);
 	}
 
+	public void function listQuestionResponsesForAjaxDataTables( event, rc, prc ) {
+		var questionId                = ( rc.questionId ?: "" );
+
+		var sFilterExpression     = ( structKeyExists( rc, 'sFilterExpression' ) && Len( Trim( rc.sFilterExpression ) ) ) ? rc.sFilterExpression : "";
+		var savedFilterExpIdLists = ( structKeyExists( rc, 'sSavedFilterExpressions' ) && Len( Trim( rc.sSavedFilterExpressions ) ) ) ? rc.sSavedFilterExpressions : "";
+
+		if ( !Len( Trim( questionId ) ) ) {
+			event.adminNotFound();
+		}
+		var canDelete       = false;
+		var useMultiActions = false;
+		var checkboxCol     = [];
+		var optionsCol      = [];
+		var gridFields      = [ "submission", "question", "submitted_by", "datecreated", "response", "is_website_user", "is_admin_user", "datecreated", "form_name" ];
+		var dtHelper        = getModel( "JQueryDatatablesHelpers" );
+		var results         = formbuilderService.getQuestionResponsesForGridListing(
+			  questionId                = questionId
+			, startRow              = dtHelper.getStartRow()
+			, maxRows               = dtHelper.getMaxRows()
+			, orderBy               = dtHelper.getSortOrder()
+			, searchQuery           = dtHelper.getSearchQuery()
+			, sFilterExpression     = sFilterExpression
+			, savedFilterExpIdLists = savedFilterExpIdLists
+		);
+		var records = Duplicate( results.records );
+		var viewQuestionResponseTitle   = "Make this i18n";//translateResource( "formbuilder:view.submission.modal.title" );
```

Comment:
What is "Make this i18n"? Should this have been replaced by an i18n value?

Also, remove commented-out code.

Reply:
Replaced with i18n and comment removed